### PR TITLE
AB#135356: Move over RQuest polling functionality from old HutchManager

### PR DESCRIPTION
**Pull request by vpnu**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] ⚡️ Optimization
- [ ] 📝 Documentation Update

## Description

Move over RquestPollingHostedService functionality from old HutchManager
Add API config for rquest calls

## Added/updated tests?

- [ ] ✅ Yes
- [x] ❌ No, and this is why:
  - _please replace this line with details on why tests have not been included_
- [ ] ❓ I need help with writing tests

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Review by dcl10 — APPROVED** [review]

File: app/RquestBridge/Config/RQuestOptions.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+namespace RquestBridge.Config;
+
+public class RQuestOptions
+{
+  public int Id { get; set; }
```
Comment (beforan) [inline app/RquestBridge/Config/RQuestOptions.cs:5]: what is this ID, is it just left over from ActivitySource or is it actually meaningful?

File: app/RquestBridge/Config/RQuestOptions.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  public string Host { get; set; } = string.Empty;
+  
+  public string ResourceId { get; set; } = string.Empty;
```
Comment (beforan) [inline app/RquestBridge/Config/RQuestOptions.cs:9]: you can rename this CollectionId now we're back in an RQuest specific world

**Review by beforan — CHANGES_REQUESTED** [review]

File: app/RquestBridge/Config/RQuestOptions.cs (hunk 1 above)
Comment (vpnu) [reply app/RquestBridge/Config/RQuestOptions.cs:5]: I left it in since I thought we might want to have one still to link to the job coming in from rquest

Comment (beforan) [reply app/RquestBridge/Config/RQuestOptions.cs:5]: sure, but that wouldn't be here in the application's config? otherwise you'd only ever have 1 job id?

Comment (vpnu) [reply app/RquestBridge/Config/RQuestOptions.cs]: cool,I removed the "activitySource" id from the RquestJob as well. We still save the rquest uuid so we can use that jobId

**Review by beforan — APPROVED** [review]

**Pull request merged**